examples/gfx/opengl/5_opengl_flappy_bird/core/batch.cpp

```diff
@@ -68,17 +68,12 @@ namespace game::core {
 
             shader_program->enable();
 
-            Mat4f model = Mat4f::identity();
-            model = model * model.scale(1.0f);
-            model = model * bebone::core::trait_bryan_angle_yxz(Vec3f(0.0f, 0.0f, 0.0f));
-            model = model * model.translation(Vec3f(0.0f, 0.0f, 0.0f));
-
-            for (auto texture : textures_to_draw) {
+            for (const auto& texture : textures_to_draw) {
                 texture->bind_texture_unit(cached_texture_units[texture]);
             }
 
             vao->bind();
-                shader_program->set_uniform("u_Model", model);
+                shader_program->set_uniform("u_Model", Mat4f::identity());

```

Comment:
btw, if  you setting this uniform as identity matrix, is there even need for it ? I mean from shader perspective, this is a constant identity matrix. As I remember you are baking all model transformations into vertex data

Reply:
Aha, ok. I will test it and make changes if needed

Reply:
I think we can't remove this line of code
![image](https://github.com/user-attachments/assets/8473a5ad-a8b5-423b-abfb-535e35f27915)

Reply:
Do not forget to clean shader 
![image](https://github.com/user-attachments/assets/91bf663c-d023-4601-b0e4-bd238b6bf352)

> as I said u_Model always identity matrix, so there is no need for multiplying vertex position by it